File: packages/python/examples/background_removal.py
import asyncio
import os
from ailabtools import AILabClient


async def main():
    client = AILabClient(api_key=os.environ["AILAB_API_KEY"])
    with open("./photo.jpg", "rb") as image:
        result = await client.background.remove({"image": image, "returnForm": "whiteBK"})
    print(result["data"]["image_url"])
    await client.aclose()


asyncio.run(main())

File: packages/node/examples/background-removal.ts
import { readFileSync } from "node:fs";
import { AILabClient } from "../src";

const client = new AILabClient({ apiKey: process.env.AILAB_API_KEY! });

async function main() {
  const result = await client.background.remove({
    image: readFileSync("./photo.jpg"),
    returnForm: "whiteBK",
  });

  console.log(result.data?.image_url);
}

main().catch((error) => {
  console.error(error);
  process.exit(1);
});
